### PR TITLE
Create General/Modulo resolves 338

### PR DESCRIPTION
**Pull request by SSwiniarski**
### Description

Crate general/modulo entry.

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.

### CODE REVIEW

File: content/general/concepts/modulo/modulo.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+Title: 'Modulo'
+Description: 'A modulo operation takes two arguments and returns the remainder when the first argument is divided by the second.'
```
Comment (yangc95) [inline content/general/concepts/modulo/modulo.md:3]:
```suggestion
Description: 'A modulo operation takes two arguments and returns the remainder of the first argument divided by the second.'
```

File: content/general/concepts/modulo/modulo.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Arithmetic'
+CatalogContent:
+  - 'paths/code-foundations'
+  - 'paths/computer-science'
+---
+
+Modulo is a mathematical operation available in every programming language along with more common ones like addition and multiplication. A modulo operation takes two arguments and returns the remainder when the first argument is divided by the second. The result of the operation is called the modulus.
```
Comment (yangc95) [inline content/general/concepts/modulo/modulo.md:14]:
```suggestion
Modulo is a mathematical operation available in every programming language along with more common ones like addition and multiplication. A modulo operation takes two arguments and returns the remainder of the first argument divided by the second. The result of the operation is called the modulus.
```

File: content/general/concepts/modulo/modulo.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+Most programming languages use the symbol `%` to represent the modulo operation, though some may use `mod`.
```
Comment (yangc95) [inline content/general/concepts/modulo/modulo.md:16]:
```suggestion
Most programming languages use the symbol `%` to represent the modulo operation, though some may use `mod` or other variations.
```

File: content/general/concepts/modulo/modulo.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+## Examples
```
Comment (yangc95) [inline content/general/concepts/modulo/modulo.md:18]: I think we should combine all the examples below this code block and pick an example programming language. 

```codebyte/js
let modulus = 14 % 5
console.log(`Results in a modulus of ${modulus}`)

modulus = 37 % 3
console.log(`Results in a modulus of ${modulus}`)

modulus = 21 % 7
console.log(`Results in a modulus of ${modulus}`)
```

Let me know what you think! Also wouldn't hurt to get a second opinion.

**Review by yangc95 — COMMENTED** [review]
Finished with the first review @SSwiniarski. Thanks for pumping out these entries! 👩‍💻

**Comment by SSwiniarski** [comment]
@yangc95, implemented your suggestions and ready for review two.

File: content/general/concepts/modulo/modulo.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+# Output : 14 % 5 results in a modulus of 4
+
+modulus = 37 % 3
+print(f"37 % 3 Results in a modulus of {modulus}")
+# Output: 37 % 3 Results in a modulus of 1
+
+modulus = 21 % 7
+print(f"21 % 7 results in a modulus of {modulus}")
+# Output: 21 % 7 results in a modulus of 0
+```
```
Comment (yangc95) [inline content/general/concepts/modulo/modulo.md:34]:
```suggestion
```codebyte/py
modulus = 14 % 5
print(f"14 % 5 results in a modulus of {modulus}")

modulus = 37 % 3
print(f"37 % 3 Results in a modulus of {modulus}")

modulus = 21 % 7
print(f"21 % 7 results in a modulus of {modulus}")
```
```

File: content/general/concepts/modulo/modulo.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+Modulo is a mathematical operation available in every programming language along with more common ones like addition and multiplication. A modulo operation takes two arguments and returns the remainder of the first argument divided by the second. The result of the operation is called the modulus.
+
+Most programming languages use the symbol `%` to represent the modulo operation, though some may use `mod` or other variations.
+
+## Examples
+
+The following is an example of the modulo operator in Python:
+
+```py
```
Comment (yangc95) [inline content/general/concepts/modulo/modulo.md:22]:
```suggestion
```codebyte/py
```

Comment (yangc95) [reply content/general/concepts/modulo/modulo.md:22]: I'm thinking we can turn this into a code byte so we wouldn't need the commented outputs

**Review by yangc95 — APPROVED** [review]

**Comment by Dusch4593** [comment]
Hey! I'm happy to help out and conduct review 2️⃣ for this one. 😄

File: content/general/concepts/modulo/modulo.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+Tags:
+  - 'Arithmetic'
+CatalogContent:
+  - 'paths/code-foundations'
+  - 'paths/computer-science'
+---
+
+Modulo is a mathematical operation available in every programming language along with more common ones like addition and multiplication. A modulo operation takes two arguments and returns the remainder of the first argument divided by the second. The result of the operation is called the modulus.
+
+Most programming languages use the symbol `%` to represent the modulo operation, though some may use `mod` or other variations.
```
[the remark this answers is not in the document]
Comment (Dusch4593) [reply content/general/concepts/modulo/modulo.md:16]: And then I would add this to line 20:

```pseudo
remainder = dividend % divisor
```

File: content/general/concepts/modulo/modulo.md (hunk 6 above)
Comment (Dusch4593) [reply content/general/concepts/modulo/modulo.md:22]: I would also include one breaking example, if possible. Like, what would we get if we tried to mod a negative integer?

```
7 % -2
```

File: content/general/concepts/modulo/modulo.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
+---
+Title: 'Modulo'
+Description: 'A modulo operation takes two arguments and returns the remainder of the first argument divided by the second.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
```
Comment (Dusch4593) [inline content/general/concepts/modulo/modulo.md:6]:
```suggestion
  - 'Code Foundations'
  - 'Computer Science'
```

File: content/general/concepts/modulo/modulo.md [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+Tags:
+  - 'Arithmetic'
```
Comment (Dusch4593) [inline content/general/concepts/modulo/modulo.md:8]:
```suggestion
  - 'Arithmetic'
  - 'Numbers'
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@SSwiniarski Finished! Left a few comments/suggestions.

**Comment by SSwiniarski** [comment]
Applied suggestions. Should be ready to go.

**Review by Dusch4593 — APPROVED** [review]
LGTM! 👍

**Comment by Dusch4593** [comment]
cc. @SSwiniarski @yangc95 

We should be good to go for merge! Just wanted to make one small edit in f9b220454ede054ee49323e455676bef44e56828 to link to the Python topic entry in Docs. This should probably be a thing for General entries going forward if they happen to use language-specific examples.

**Pull request merged**